### ISSUE TITLE
ValueError: (False, 'Unknown discrete action jumpeast')

### ISSUE THREAD

**Issue by figgybit** [tambetm/gym-minecraft#6]
```
import gym
import gym_minecraft
env = gym.make('MinecraftCliffWalking1-v0')

env.init(start_minecraft=True)

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/jonkowalski/Desktop/MOVEIT/Desktop/Personal/SCHOOL/GATECH/7641/virtual_ml/venv/lib/python2.7/site-packages/gym_minecraft-0.0.2-py2.7.egg/gym_minecraft/envs/minecraft_env.py", line 129, in init
    self._create_action_space()
  File "/Users/jonkowalski/Desktop/MOVEIT/Desktop/Personal/SCHOOL/GATECH/7641/virtual_ml/venv/lib/python2.7/site-packages/gym_minecraft-0.0.2-py2.7.egg/gym_minecraft/envs/minecraft_env.py", line 191, in _create_action_space
    raise ValueError(False, "Unknown discrete action " + cmd)
ValueError: (False, 'Unknown discrete action jumpeast')

```

when debugging the code there seems to be a large set of discrete actions that are not supported

```
(Pdb) cmds = self.mission_spec.getAllowedCommands(0, ch)
(Pdb) [x for x in cmds]
['jump', 'jumpeast', 'jumpmove', 'jumpnorth', 'jumpsouth', 'jumpstrafe', 'jumpuse', 'jumpwest', 'look', 'move', 'moveeast', 'movenorth', 'movesouth', 'movewest', 'strafe', 'turn', 'use']
```

https://github.com/tambetm/gym-minecraft/blob/master/gym_minecraft/envs/minecraft_env.py#L182-L191

**Comment by tambetm** [comment]
Oh, it seems that newer versions have added `jump*` actions. I'd be happy to accept PR on this, possibly those new actions just need to be added to [line 183](https://github.com/tambetm/gym-minecraft/blob/master/gym_minecraft/envs/minecraft_env.py#L183).

**Comment by figgybit** [comment]
OK I will do it.

**Comment by figgybit** [comment]
I tried to branch your project, commit my change, and create a pull request.  

However:
fatal: unable to access 'https://github.com/tambetm/gym-minecraft.git/': The requested URL returned error: 403

So I forked your project and committed over here. 
https://github.com/figgybit/gym-minecraft/commit/467ae4d1a74014a3218340cfe69908aff3c23cdf

Any advice on being able to push my branch would be appreciated. 
Thanks
